conf/config.sh.dist

```diff
@@ -170,3 +170,6 @@ DB_AUTH_NAME="acore_auth"
 DB_CHARACTERS_NAME="acore_characters"
 
 DB_WORLD_NAME="acore_world"
+
+# Branch configuration for installer to pull from
+INSTALLER_PULL_FROM="master"
```

Comment:
Since we have several branches and someone could clone -b "2.x" for example.
For a future-proof configuration would be better to write something like:

```
# Branch configuration for the installer to pull from.
# By default git will select the current working branch
# You can change it to "master" if you want the latest updates
INSTALLER_PULL_FROM=$(git rev-parse --abbrev-ref HEAD)
```